k2/csrc/properties_test.cc

```diff
@@ -138,20 +174,47 @@ TEST(Properties, IsNotEpsilonFree) {
 }
 
 TEST(Properties, IsEpsilonFree) {
-  std::vector<Arc> arcs = {
-      {0, 1, 2},
-      {0, 2, 1},
-      {1, 2, 1},
-  };
-  std::vector<Range> leaving_arcs = {
-      {0, 2},
-      {2, 3},
-  };
+  std::vector<Arc> arcs = {{0, 1, 2}, {0, 2, 1}, {1, 2, 1}, };
```

Comment:
what is your clang-format version?
It seems that different clang-format versions have different behaviors.

Reply:
Thanks @csukuangfj , here is my version.
```
(py-kaldi) [qiuhaowen@c3-asr-dev-train-i2-02 build]$ clang-format --version
LLVM (http://llvm.org/):
  LLVM version 3.4.2
  Optimized build.
  Built Nov  1 2018 (15:06:24).
  Default target: x86_64-redhat-linux-gnu
  Host CPU: core-avx2

```